invenio_app_rdm/communities_ui/views/communities.py

```diff
@@ -30,8 +30,7 @@ def communities_detail(pid_value, community, community_ui):
     return render_community_theme_template(
         "invenio_communities/records/index.html",
         theme=community_ui.get("theme", {}),
-        community=community,
-        community_ui=community_ui,
+        community=community_ui,
```

Comment:
By all accounts that was a bug given everyother related views maps `community` to `community_ui`.